### ISSUE TITLE
When NVDA is on, user must press Esc twice to close dropdown.

### ISSUE THREAD

**Issue by mvaibhav96** [chrmarti/testissues#28831]
Repro Steps:

1. Open the attached HTML page.
2. Start NVDA and navigate to the "Dropdown button".
3. Navigate till anyone of the list item and press escape.

Actual Result:-
When NVDA is on user need to press Esc two times to close dropdown.

Expected Result:-
User should be able to close the list items on pressing the dropdown only once. Command to perform any operation should be changed when NVDA is on.

User Impact:-
Screen Reader users find difficulty in using application if user needs to operate controls twice to perform an action.

[Sample HTML page.zip](https://github.com/chrmarti/testissues/files/2433245/Sample.HTML.page.zip)
